dvc/stage/cache.py

```diff
@@ -80,19 +80,8 @@ def _load(self, stage):
     def _create_stage(self, cache, wdir=None):
         from dvc.stage import create_stage, PipelineStage
 
-        params = []
-        for param in cache.get("params", []):
-            if isinstance(param, str):
-                params.append(param)
-                continue
-
-            assert isinstance(param, dict)
-            assert len(param) == 1
-            path = list(param.keys())[0]
-            params_list = param[path]
-            assert isinstance(params_list, list)
-            params.append(f"{path}:" + ",".join(params_list))
-
+        params_cache = cache.get("params", {})
+        params = [{f: list(p.keys())} for f, p in params_cache.items()]
```

Comment:
@efiop, is params and deps even required here (I assume not, but just wanted to confirm)?

Reply:
You are right, this is not used right now. We are filling everything from cached locks anyway, so this is not used. Let's get rid of it.

Reply:
Removed.